src/workspacer.Bar/BarPluginConfig.cs

```diff
@@ -12,7 +12,6 @@ public class BarPluginConfig
 
         public Color DefaultWidgetForeground { get; set; } = Color.White;
         public Color DefaultWidgetBackground { get; set; } = Color.Black;
-        public Color Background { get; set; } = Color.Black;
```

Comment:
Any reason why this property has been removed?

Reply:
Mainly due to being unused, as `DefaultWidgetBackground` is the property used for the form background at the moment.

Reply:
Fair enough. Then we keep it.